scripts/generate-integration-pages.ts

```diff
@@ -178,6 +179,47 @@ function absoluteLinks({ base }: { base: string }) {
 	};
 }
 
+/** Remark plugin to replace GitHub . */
+function replaceAsides() {
+	return function transform(tree: Root) {
+		// visit blockquote
+		visit(tree, 'blockquote', (node) => {
+			// first child (<blockquote><p>...</p></blockquote>)
+			const [firstChild, trailingText] = node.children[0].children;
+
+			if (firstChild.type === 'strong') {
+				if (
+					// check for **Note** or **Warning**
+					firstChild.children[0].value.includes('Note') ||
+					firstChild.children[0].value.includes('Warning')
+				) {
+					const AsideType =
+						firstChild.children[0].value.toLowerCase() === 'note' ? 'note' : 'caution';
+
+					// remove blockquotes `>`
+					node.type = 'paragraph';
+
+					// if starts with `: ` remove it
+					if (trailingText.value.startsWith(': ')) {
+						trailingText.value = trailingText.value.replace(/^: /, '\n');
+					}
```

Comment:
Should be safe to skip the `if` here I think: if `/^: /` doesn’t match, nothing will be replaced anyway.

```suggestion
					trailingText.value = trailingText.value.replace(/^: /, '\n');
```

Reply:
Question: do we need to add the additional `\n`? If so, I’d love the comment in the line above to mention why.